frodo.js

```diff
@@ -1,63 +1,66 @@
-const evenNumbers = numbers.filter(num => num % 2 === 0);
```

Comment:
SCRIPT-This hardcoded value might be better suited as a constant at the top of the file.